modules/internal/DefaultAssociative.chpl

```diff
@@ -147,7 +147,51 @@ module DefaultAssociative {
         }
       }
     }
-  
+ 
+    iter these(param tag: iterKind) where tag == iterKind.standalone {
+      if debugDefaultAssoc then
+        writeln("*** In associative domain standalone iterator");
+      const numTasks = if dataParTasksPerLocale==0 then here.maxTaskPar
+                       else dataParTasksPerLocale;
+      const ignoreRunning = dataParIgnoreRunningTasks;
+      const minIndicesPerTask = dataParMinGranularity;
+      // We are simply slicing up the table here.  Trying to do something
```

Comment:
Sorry, I should have been more clear with my comment: I didn't mean for this PR, it was just wondering after reading the comment in the code.

I think you're right that it should stay simple for this PR